From 9f2cf209ef8235f33683ca941fd0d7bdcca4078d Mon Sep 17 00:00:00 2001
From: Matt Porter <mporter@konsulko.com>
Date: Sun, 17 Dec 2017 21:44:41 -0500
Subject: [PATCH] arm64: dts: renesas: disable r8a7796 hscif0 dma

When streaming A2DP audio on WiLink8, DMA errors are
seen from the rcar-dmac driver servicing hscif0. As
a result, data corruption is seen by the UIM service
managing the link to the Wilink8 device. This data
corruption manifests as dropped frames and audibly as
"skips" in the audio playback.

Workaround this issue by disabling DMA for hscif0.

Signed-off-by: Matt Porter <mporter@konsulko.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index e3a5c25bc3da..dd97af4bdd60 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -1282,7 +1282,6 @@
 				 <&scif_clk>;
 			clock-names = "fck", "brg_int", "scif_clk";
 			dmas = <&dmac1 0x31>, <&dmac1 0x30>;
-			dma-names = "tx", "rx";
 			power-domains = <&sysc R8A7796_PD_ALWAYS_ON>;
 			status = "disabled";
 		};
-- 
2.11.0

